core/drivers/pcap_test.cc

```diff
@@ -0,0 +1,131 @@
+#include <gtest/gtest.h>
+#include <google/protobuf/util/message_differencer.h>
+#include "pcap.h"
+
+#define TEST_STRLEN 25
```

Comment:
Let's use std::string. In general we do not add C macros unless absolutely necessary.